aroquev/src/main/webapp/script.js

```diff
@@ -31,45 +30,61 @@ function addRandomGreeting() {
  * Adds Armandos age to the page.
  */
 function calculateArmandoAge() {
-    // Calculate age, Armando was born on May 13, 2000.
-    const diffMs = new Date() - new Date(2000, 5, 13);
-    const ageDt = new Date(diffMs); 
-    
-    // Add it to the page.
-    const ageContainer = document.getElementById('armando-age-container');
-    ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
-    
+  // Calculate age, Armando was born on May 13, 2000.
+  const diffMs = new Date() - new Date(2000, 5, 13);
+  const ageDt = new Date(diffMs); 
+  
+  // Add it to the page.
+  const ageContainer = document.getElementById('armando-age-container');
+  ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);   
 }
 
+calculateArmandoAge();
+
 // Scripts for the image gallery
 var slideIndex = 1;
 
 function initSlides() {
   showSlides(slideIndex);
 }
 
-// Next/previous controls
-function plusSlides(n) {
-  showSlides(slideIndex += n);
+// Next control
+function showNextSlide() {
+	slideIndex ++;
```

Comment:
nit. remove whitespace between `slideIndex` and `++` operator. Same comment for below with `--`.

Reply:
Done